examples/raw_calculations/kagome_antiferro.py

```diff
@@ -45,6 +45,7 @@ def kagome_antiferromagnet(n_q = 100, classes = py_classes):
         rotation(-2 * np.pi / 3)
     ]
     magnitudes = np.array([1.0]*3)  # spin-1
+    positions = [np.array([0., 0., 0.]), np.array([1., 0., 0.,]), np.array([0., 1., 0.])]
```

Comment:
Maybe put this on three lines like the kagome ferromagnet example above for consistency?